File: src/main/java/com/dyonovan/researchsystem/common/container/ContainerResearchManager.java
package com.dyonovan.researchsystem.common.container;

import com.dyonovan.researchsystem.common.tileentity.TileResearchManager;
import net.minecraft.entity.player.EntityPlayer;
import net.minecraft.entity.player.InventoryPlayer;
import net.minecraft.inventory.Container;

/**
 * This file was created for Research System
 * <p>
 * Research System is licensed under the
 * Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License:
 * http://creativecommons.org/licenses/by-nc-sa/4.0/
 *
 * @author Dyonovan
 * @since 6/24/2016
 */
public class ContainerResearchManager extends Container {

    public ContainerResearchManager(InventoryPlayer inventory, TileResearchManager tileEntity) {

    }

    @Override
    public boolean canInteractWith(EntityPlayer playerIn) {
        return true;
    }
}

File: src/main/scala/com/dyonovan/researchsystem/common/container/ContainerResearchManager.scala
package com.dyonovan.researchsystem.common.container

import com.dyonovan.researchsystem.common.tileentity.TileResearchManager
import net.minecraft.entity.player.{EntityPlayer, InventoryPlayer}
import net.minecraft.inventory.Container

/**
  * This file was created for ResearchSystem
  *
  * ResearchSystem is licensed under the
  * Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License:
  * http://creativecommons.org/licenses/by-nc-sa/4.0/
  *
  * @author Dyonovan
  * @since 7/22/2016
  */
class ContainerResearchManager(playerInventory: InventoryPlayer, tile: TileResearchManager) extends Container {




  override def canInteractWith(playerIn: EntityPlayer): Boolean = true
}
